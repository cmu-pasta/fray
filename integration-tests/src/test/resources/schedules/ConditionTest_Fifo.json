{
    "choices": [
        {
            "selected": 0,
            "threadId": 0,
            "enabled": 1
        },
        {
            "selected": 0,
            "threadId": 0,
            "enabled": 1
        },
        {
            "selected": 0,
            "threadId": 0,
            "enabled": 2
        },
        {
            "selected": 0,
            "threadId": 0,
            "enabled": 3
        },
        {
            "selected": 0,
            "threadId": 0,
            "enabled": 3
        },
        {
            "selected": 0,
            "threadId": 1,
            "enabled": 2
        },
        {
            "selected": 0,
            "threadId": 1,
            "enabled": 2
        },
        {
            "selected": 0,
            "threadId": 1,
            "enabled": 2
        },
        {
            "selected": 0,
            "threadId": 1,
            "enabled": 2
        },
        {
            "selected": 0,
            "threadId": 2,
            "enabled": 1
        },
        {
            "selected": 0,
            "threadId": 2,
            "enabled": 1
        },
        {
            "selected": 0,
            "threadId": 1,
            "enabled": 2
        },
        {
            "selected": 0,
            "threadId": 1,
            "enabled": 2
        },
        {
            "selected": 0,
            "threadId": 1,
            "enabled": 2
        },
        {
            "selected": 0,
            "threadId": 2,
            "enabled": 1
        },
        {
            "selected": 0,
            "threadId": 1,
            "enabled": 2
        },
        {
            "selected": 0,
            "threadId": 0,
            "enabled": 2
        },
        {
            "selected": 0,
            "threadId": 0,
            "enabled": 2
        },
        {
            "selected": 0,
            "threadId": 0,
            "enabled": 2
        },
        {
            "selected": 0,
            "threadId": 2,
            "enabled": 1
        },
        {
            "selected": 0,
            "threadId": 0,
            "enabled": 1
        }
    ],
    "fullSchedule": true
}