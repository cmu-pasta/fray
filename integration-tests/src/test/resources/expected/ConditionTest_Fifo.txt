[1]: Acquired lock.
[1]: Sent full signal.
[1]: Released lock.
[1]: Acquired lock.
[1]: Beginning empty await.
[2]: Acquired lock.
[2]: Sent empty signal.
[2]: Released lock.
[1]: Returned from empty await.
[1]: Sent full signal.
[1]: Released lock.
[1]: Acquired lock.
[1]: Beginning empty await.
[2]: Acquired lock.
[2]: Sent empty signal.
[2]: Released lock.
[1]: Returned from empty await.
[1]: Sent full signal.
[1]: Released lock.
[2]: Acquired lock.
[2]: Sent empty signal.
[2]: Released lock.

